### PR TITLE
Add option to reload templates

### PR DESCRIPTION
**Pull request by charmander**
For your development convenience. Maybe there should be a `sass:watch` task at some point, too.

### CODE REVIEW

File: weasyl/define.py [hunk 1]
```diff
@@ -204,21 +213,21 @@ def sql_string_series(target):
...
-    else:
-        template = template_name
-    if template is None:
+def compile(template_name):
+    """
+    Compiles a template file and returns the result.
+    """
+    template = _template_cache.get(template_name)
+
+    if template is None or reload_templates:
```
Comment (weykent) [inline weasyl/define.py]: Considering template crap uses such a tiny amount of CPU it seems like you don't need anything more complex than this. Great! I don't know why this didn't get added earlier.

File: weasyl/define.py [hunk 2]
```diff
@@ -1081,6 +1088,9 @@ def cdnify_url(url):
 
 
 def get_resource_path(resource):
+    if reload_templates:
```
Comment (weykent) [inline weasyl/define.py]: Maybe 'reload templates' is too specific if it's going to be used here too?

Comment (charmander) [inline weasyl/define.py]: Maybe, but I couldn’t come up with a better name. What describes the templates and the resources but not the non-template Python?

Comment (weykent) [inline weasyl/define.py]: Assets? That might be too specific to non-templates. It looks like pyramid separately allows reloading of 'templates' and 'assets'. Is `reload_templates_and_assets` too verbose?

Comment (charmander) [inline weasyl/define.py]: I can add a separate `reload_assets`.

**Pull request merged**